pkgs/development/compilers/go/1.16.nix

```diff
@@ -45,7 +45,7 @@ stdenv.mkDerivation rec {
 
   src = fetchurl {
     url = "https://dl.google.com/go/go${version}.src.tar.gz";
-    sha256 = "1sl33wkhp6pi9f15f6khp5a7l7xwmpc3sp1zmji8pjr3g8l19jip";
+    sha256 = "sha256-spjSnekjbKR6Aj44IxO8wtLu0x36cGtgoEEDzoOnGiU=";
```

Comment:
Why is the format changed here?

Reply:
`nixpkgs-update` generates the `sha256-` prefixed hashes

Reply:
Thats not quite right. nix-update uses the hash the nix-daemon provides which is sri variant for nixUnstable. If you are using nixStable then you get the other format. I think you need to manually convert it if you want the other.